### PR TITLE
prepare 0.8.0 release

### PR DESCRIPTION
**Pull request by ciyer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+
+**project**
+
+* Close the fork dialog after forking [#723](https://github.com/SwissDataScienceCenter/renku-ui/issues/723), [#745](https://github.com/SwissDataScienceCenter/renku-ui/pull/745)
+
+
+### BREAKING CHANGES
+
+* Requires renku-gateway 0.7.0
+* Requires renku-graph 0.47.1
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.md]: I believe it's `0.47.2` here

Comment (ciyer) [reply CHANGELOG.md]: Thanks for the clarification! I have made this correction.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,60 @@
 # Changes
 
+## [0.8.0](https://github.com/SwissDataScienceCenter/renku-ui/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A0.8.0) (released 2020-03-04)
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.md]: The code seems correct here but the preview doesn't show me the `0.8.0` text :thinking: 
Screenshot_20200304_113642

Comment (ciyer) [reply CHANGELOG.md]: I copy/pasted that into the description of the release https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/untagged-8991d4bfe686f8dcba65 and it renders correctly there.

Comment (lorenzo-cavazzi) [reply CHANGELOG.md]: ok great

**Comment by ciyer** [comment]
> Should we mention also this change in the "bug fixes"? #782
> The users will be impacted by possibly seeing this when opening the UI
> 
> 
> Maybe something like "User profile is checked before showing the UI"

Done.

**Pull request merged**